SECURITY-INSIGHTS.yml

```diff
@@ -62,3 +62,5 @@ dependencies:
     sbom-url: https://github.com/anchore/sbom-action 
   dependencies-lifecycle:
     policy-url: https://github.com/jaegertracing/jaeger/blob/main/SECURITY.md#security-patch-policy
+  env-depedencies-policy:
+    policy-url: https://github.com/jaegertracing/jaeger/blob/main/SECURITY.md#dependencies-lifecycle-policy
```

Comment:
```suggestion
    policy-url: https://github.com/jaegertracing/jaeger/blob/main/SECURITY.md#dependency-policy
```

Reply:
I think the check in CLOMonitor requires they are different, but lets try it!